.gitignore

```diff
@@ -36,4 +36,6 @@ lerna-debug.log*
 !.vscode/settings.json
 !.vscode/tasks.json
 !.vscode/launch.json
-!.vscode/extensions.json
\ No newline at end of file
+!.vscode/extensions.json
+
+/packages/energy-api-app/schema.yaml
```

Comment:
`energy-api-app` has it's own .gitignore file:
https://github.com/energywebfoundation/energy-api/blob/master/packages/energy-api-app/.gitignore

This should be moved there.

Reply:
Ok